src/core/search/sagas/searchOrganizationDataSets.js

```diff
@@ -0,0 +1,116 @@
+/*
+ * @flow
+ */
+
+import { call, put, takeEvery } from '@redux-saga/core/effects';
+import { Set } from 'immutable';
+import { DataUtils, Logger, ValidationUtils } from 'lattice-utils';
+import type { Saga } from '@redux-saga/core';
+import type { UUID } from 'lattice';
+import type { WorkerResponse } from 'lattice-sagas';
+import type { SequenceAction } from 'redux-reqseq';
+
+import { searchDataWorker } from './searchData';
+import { searchDataSetsWorker } from './searchDataSets';
+
+import { getOrSelectDataSets } from '../../edm/actions';
+import { FQNS } from '../../edm/constants';
+import { getOrSelectDataSetsWorker } from '../../edm/sagas';
+import {
+  ATLAS_DATA_SET_IDS,
+  ENTITY_SET_IDS,
+  HITS,
+  TOTAL_HITS,
+} from '../../redux/constants';
+import {
+  SEARCH_ORGANIZATION_DATA_SETS,
+  searchData,
+  searchDataSets,
+  searchOrganizationDataSets,
+} from '../actions';
+
+const LOG = new Logger('SearchSagas');
+
+// TODO: DELETE ONCE PROPERLY IMPLEMENTED
+const SHIP_ROOM_ORG_ID = '81999873-5b22-434e-be9b-1f98931ae2e4';
+const SR_DS_META_ESID = '091695e1-a971-40ee-9956-a6a05c5942dd';
+
+const { getPropertyValue } = DataUtils;
+const { isValidUUID } = ValidationUtils;
+
+function* searchOrganizationDataSetsWorker(action :SequenceAction) :Saga<*> {
+
+  try {
+    yield put(searchOrganizationDataSets.request(action.id, action.value));
+
+    const organizationId :UUID = action.value.organizationId;
+    let atlasDataSetIds :Set<UUID> = Set();
+    let entitySetIds :Set<UUID> = Set();
+
+    let response :WorkerResponse;
+
+    if (organizationId === SHIP_ROOM_ORG_ID) {
+
+      response = yield call(
+        searchDataWorker,
+        searchData({
+          ...action.value,
+          entitySetId: SR_DS_META_ESID,
+        })
+      );
+      if (response.error) throw response.error;
+
+      const entitySetIdsHits = response.data.hits
+        // .filter((hit) => getPropertyValue(hit, [FQNS.OL_STANDARDIZED, 0]) === true)
```

Comment:
Why is this commented out? Even if the property doesn't exist on the set, it doesn't seem like it should affect anything. Same with the similar comment for atlasDataSetIdsHits

Reply:
Fixed.